bedrock/firefox/templates/firefox/desktop/index-b.html

```diff
@@ -0,0 +1,31 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/desktop/index.html" %}
+
+{% block email_form %}
+  <aside id="subscribe-download-wrapper" class="has-variant ga-section" data-ga-label="Get Firefox news">
+    <div class="content container">
+      <section class="variant">
+        <header>
+        {% if LANG.startswith('en-') %}
+          <h3>{{ _('Keep up with all things Firefox.') }}</h3>
+        {% else %}
+          <h3>{{ _('Get Firefox news') }}</h3>
+        {% endif %}
+        </header>
+        {{ email_newsletter_form(title=' ') }}
+      </section>
+
+      <section class="original">
+        <div id="subscribe-wrapper">
+          {{ email_newsletter_form(title=_('Get Firefox news')) }}
```

Comment:
Embedding two newsletters in the same template here is breaking the JS form submission, because it creates duplicate ID's on the same page. This is also causing [this functional test](https://github.com/mozilla/bedrock/blob/master/tests/functional/firefox/desktop/test_all.py#L24) to fail as a result of the duplicate ID's.